src/ecommerce.interfaces.ts

```diff
@@ -96,28 +96,69 @@ export interface SDKECommerceAPI extends IECommerceShared {
         customFlags?: SDKEventCustomFlags,
         eventOptions?: SDKEventOptions
     ): void;
+    setCurrencyCode(code: string): void;
+
+    /*
+     * @deprecated
+     */
     logPurchase(
         transactionAttributes: TransactionAttributes,
         product: SDKProduct | SDKProduct[],
         clearCart?: boolean,
         attrs?: SDKEventAttrs,
         customFlags?: SDKEventCustomFlags
     ): void;
+    /*
+     * @deprecated
+     */
```

Comment:
for consistency with spacing, either add a line before the deprecation on line 111 or remove the new line at 100